Source/Core/VideoBackends/D3D/Render.cpp

```diff
@@ -285,6 +289,31 @@ void Renderer::WaitForGPUIdle()
   D3D::context->Flush();
 }
 
+void Renderer::RenderXFBToScreen(const MathUtil::Rectangle<int>& target_rc,
+                                 const AbstractTexture* source_texture,
+                                 const MathUtil::Rectangle<int>& source_rc)
+{
+  if (auto* const openxr_session = GetOpenXRSession())
+  {
+    const auto image_index = openxr_session->AquireAndWaitForSwapchainImage();
+
+    if (!image_index.has_value())
+      return;
+
+    const auto images = openxr_session->GetSwapchainImages(
+        XrSwapchainImageD3D11KHR{XR_TYPE_SWAPCHAIN_IMAGE_D3D11_KHR});
+
+    ComPtr<ID3D11Texture2D> texture(images[*image_index].texture);
+
+    auto dx_texture = DXTexture::CreateAdopted(std::move(texture));
```

Comment:
These (texture/framebuffer) are unique_ptrs and should be stored somewhere. The only reason it's probably not breaking is because the internal reference count on the objects gets increased when it's bound to the device.

Reply:
Yeah. I know this code is a little gross.. but as long as the internal reference counting is defined behavior it should be fine, yeah?

Reply:
Less of an issue in DX11, more of an issue in DX12. I guess we'll have the same problem in the other backends too, so we'll need a more general solution regardless.